contribs/gmf/src/directives/search.js

```diff
@@ -228,6 +234,13 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
    */
   this.featureOverlay_ = ngeoFeatureOverlayMgr.getFeatureOverlay();
 
+  var featureStyle = this.scope_['featuresStyle'];
+  if (featureStyle instanceof ol.style.Style ||
+      featureStyle instanceof Array ||
+      featureStyle instanceof Function) {
+    this.featureOverlay_.setStyle(featureStyle);
```

Comment:
Are we sure we have only features from search in the feature overlay manager when search is active ?
Why not puting the style in the feature ?